bin/cron/teacher_applications_to_gdrive

```diff
@@ -13,7 +13,7 @@ def get_rows
   applications = [%w(course regional_partner status scholarship_status school_type pay_fee email created_at
                      meets_minimum_requirements meets_scholarship_criteria registered_for_workshop rural_status
                      free_reduced_lunch_percent urm_percent accepted_at accepted race community_type two_or_more_races)]
-  Pd::Application::TeacherApplication.find_each do |app|
+  Pd::Application::TeacherApplication.where(application_year: '2021-2022').find_each do |app|
```

Comment:
Is there a way to future proof this so that we don't have to update it each year? Or should we track it as something that needs updating each year?

Reply:
We could tie it to the `APPLICATION_CURRENT_YEAR` constant, but I'm afraid of adding additional unexpected side effects to changing that constant, which currently also archives the applications and updates the content of the application form. I think it makes sense, though, as this would be expected to update as we archive. I'll change it

Reply:
Can you note all the things that constant does in a comment near it?